### PR TITLE
Added gig diary

### PR DESCRIPTION
**Pull request by jarhodes314**

### CODE REVIEW

File: cypress/integration/public.spec.js [hunk 1]
```diff
@@ -62,20 +62,16 @@ describe("book us page", () => {
     cy.get(".testimonial").should("be.visible");
   });
 
-  it("allows a user to submit a booking request", () => {
-    cy.route2({
-      method: "POST",
-      url: "/contact",
-    }).as("contact");
+  it("allows a user to submit a booking request", { browser: ["chromium", "chrome", "electron"] }, () => {
```
Comment (caro401) [inline cypress/integration/public.spec.js:65]: why electron?

File: src/components/Gigs/Calendar.svelte [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+  <div class="right">
+    <button on:click="{() => (showSelection = !showSelection)}" title="Select month">
+      <i class="las la-calendar"></i>
+    </button>
+    <button on:click="{() => dispatchEvent('clickNext')}" data-test="gigcalendar-next-month">Next</button>
+  </div>
+</div>
+{#if showSelection}
+  <div class="month-selector">
+    <!-- svelte-ignore a11y-no-onchange -->
```
Comment (caro401) [inline src/components/Gigs/Calendar.svelte:369]: ? what does this do?

File: src/components/TooltipText.svelte [hunk 3]
```diff
@@ -1,28 +1,48 @@
...
   import "tippy.js/dist/tippy.css";
-  import { onMount } from "svelte";
-  import md5 from "md5";
+  import { onDestroy, onMount } from "svelte";
   import escapeHtml from "escape-html";
+  import { themeName } from "../view";
   export let content;
-  let id = md5(content);
+
+  function makeid(length) {
```
Comment (caro401) [inline src/components/TooltipText.svelte:9]: wat

File: src/routes/members/gigs/index.svelte [hunk 4]
```diff
@@ -1,9 +1,389 @@
...
+</div>
 
-<h1>Gigs</h1>
+{#each gigs as gig (gig.id)}
+  {#if gig.id in signupGigs && typeof signupGigs[gig.id].subscribe !== 'undefined'}
+    <Summary {gig} signupGig="{signupGigs[gig.id]}" {userInstruments} linkHeading="{true}" />
+  {:else}
+    <Summary {gig} {userInstruments} linkHeading="{true}" />
+  {/if}
+{:else}No gigs to display{/each}
```
Comment (caro401) [inline src/routes/members/gigs/index.svelte:389]: what is this next empty file for?

**Review by caro401 — DISMISSED** [review]

File: cypress/integration/public.spec.js (hunk 1 above)
Comment (caro401) [reply cypress/integration/public.spec.js:65]: default for cypress, easy in docker

File: src/components/Gigs/Calendar.svelte (hunk 2 above)
Comment (jarhodes314) [reply src/components/Gigs/Calendar.svelte:369]: Disables a warning about onchange on select, which is mostly redundant in modern browsers.

**Pull request merged**